### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 21: ordinal not in range(128)

### ISSUE THREAD

**Issue by Moumoutaru** [willthames/ansible-review#52]
Seeing this when trying to run ansible-review on a diff. Anyone see it before?

```
⇒  git diff origin branch | ansible-review -c .ansible-review.ini
Traceback (most recent call last):
  File "/usr/bin/ansible-review", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python2.7/site-packages/ansiblereview/__main__.py", line 76, in main
    candidates = get_candidates_from_diff(sys.stdin)
  File "/usr/lib/python2.7/site-packages/ansiblereview/__main__.py", line 20, in get_candidates_from_diff
    patch = unidiff.PatchSet(sys.stdin)
  File "/usr/lib/python2.7/site-packages/unidiff/patch.py", line 353, in __init__
    self._parse(data, encoding=encoding)
  File "/usr/lib/python2.7/site-packages/unidiff/patch.py", line 399, in _parse
    current_file._parse_hunk(line, diff, encoding)
  File "/usr/lib/python2.7/site-packages/unidiff/patch.py", line 267, in _parse_hunk
    hunk.append(original_line)
  File "/usr/lib/python2.7/site-packages/unidiff/patch.py", line 165, in append
    s = str(line)
  File "/usr/lib/python2.7/site-packages/unidiff/patch.py", line 58, in <lambda>
    cls.__str__ = lambda x: x.__unicode__().encode(DEFAULT_ENCODING)
  File "/usr/lib/python2.7/site-packages/unidiff/patch.py", line 86, in __str__
    return "%s%s" % (self.line_type, self.value)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 21: ordinal not in range(128)
```

**Comment by Moumoutaru** [comment]
https://github.com/willthames/ansible-review/blob/master/lib/ansiblereview/__main__.py#L20

**Comment by Moumoutaru** [comment]
This issue was closed by 54d660c259c8a7176fecb598a04b1613840644ec
